src/pages/UserPage/classes.module.scss

```diff
@@ -39,18 +39,35 @@
   top: spacing(1);
 }
 
-.deactivateContent > div > div {
-  color: $color-primary;
-  > form {
-    padding-top: spacing(6);
-  }
+.deactivateContent > div > div form {
+  padding-top: spacing(6);
 }
 
 :global(.react-datepicker__tab-loop) {
   position: fixed;
 }
 
+.deactivateContent > div > div > div,
+.activateContent > div > div > div {
+  color: $color-primary;
+}
+
+.activateContent h1,
 .deactivateContent h1,
+.activateContent > div > div,
 .deactivateContent > div > div {
   max-width: 70%;
 }
+
+.activateContent > div > div > form {
```

Comment:
**[blocker]**

Might not be able to get rid of all of these, but let's try to reduce the number of these types of style overwrites where possible. They work, but they are also really error prone, since they depend on the exact structure of html and will break, when that structure changes even a bit.

For example you could pass className prop to the DynamicForm directly, instead of the ".activateContent > div > div > form " targeting

Didnät go over the other options here, but I'm guessing you could get rid of some others as well. (For example the :first-child selector below seems to be targeting one of the inputs inside the form, so we should instead pass a custom className to that input)